### ISSUE TITLE
Just list of things that annoys me alot - tier 2

### ISSUE THREAD

**Issue by Cinobi** [GTNewHorizons/NewHorizons#2954]
#### Which modpack version are you using?
Epsi one
#
#### If in multiplayer; On which server does this happen? 
Epsi live server
#
....
9. Stemcells chanse from eggs too low, buff it please or reduse time needed to disassible eggs.
10. I hear some rumors that circuit assemblers in processing arrays can work out of cleanroom. Not tested myself, though.
....

And now, not serios stuff:
- u can add some visual effets for player-nicks who craft some crazy shit(infinity chest for example) or u can add some quests that's will give medals for players which have some specific shit(like full infinity armor+infinity sword)
- some of my baddyes want to see next rank after the elite with some special abilities like multicolor shit, some special abilities, some realy neat quests in coins-coins-coins(bying stack of chunkload coins for infi ingot for example)
- add Herobrine, please!

------------------------------------------------
most of issues resended to they personal place~

**Comment by noobyaran9** [comment]
7. is not possible. DE is discontinued on 1.7.10.

**Comment by Namikon** [comment]
That never stopped us to change stuff, did it?
But I guess any Server will start to suffer while you try to mine 1300+ Blocks.
If this becomes a problem in the future, we'll have to reduce/disable the depth modifier on that pick

**Comment by Cinobi** [comment]
@Namikon the problem not in 1300+ blocks. Bloodmagic bound pickaxe mine the same area, but couse no tps drop - it uses different mechanic of block breaking. It just remove blocks and add them to inventory, and draconic stuff simulates that u break each block by urself, just realy speedy. This can be seen in lags - blocks dissapearing one by one in lines.

**Comment by noobyaran9** [comment]
@Cinobi That would require a code change which means they either need to fork the mod or reduce the depth and range.

**Comment by Cinobi** [comment]
@noobyaran9 it couse tps drop even with low amount of range\depth. Aaand, nerf endgame items to shit is strange solution, don't u think?

**Comment by noobyaran9** [comment]
>draconic stuff simulates that u break each block by urself, just realy speedy. This can be seen in lags - blocks dissapearing one by one in lines.

Requires code change to fix this.

**Comment by Cinobi** [comment]
> Requires code change to fix this.
Oh, realy? wow, so fresh info, never think about it! wow wow, wow!

**Comment by Namikon** [comment]
No need to rage on that now. We're probably one of the only few modpacks who really change other mods by code and not just throw in new recipes and/or quests.

The other Goal of us is to keep the Servers playable; As much as it is possible. And I have already observed heavy lag spikes with people mining like that.

TL;DR:
If no one volunteers to fork, modify and maintain a custom DE, the only solution for this is to nerf/disable these Items; So other people will not have these lag spikes.

**Comment by Dream-Master** [comment]
de is forked allready we using custom version

**Comment by Namikon** [comment]
Then maybe someone will change it.

@Cinobi Please make individual issues. These "20 in 1" issues are impossible to keep track of

**Comment by Dream-Master** [comment]
i close this now. All issues are posted allready.
